### ISSUE TITLE
Fatal error on every Graphql request

### ISSUE THREAD

**Issue by daiduongvo18** [mollie/magento2#262]
Hello,

I have a error with GraphQl API after upgrading the extension to new version.

Thank you for checking.

**Describe the bug**
A fatal error occurs on every GraphQl request. The root cause is the class `\Magento\Framework\GraphQl\Config\Element\Field` in `vendor/mollie/magento2/LegacySupport/GraphQl/Field.php` doesn't implement `Magento\Framework\GraphQl\Config\Element\FieldInterface`. Although there is a check to ensure that file `vendor/mollie/magento2/LegacySupport/GraphQl/Field.php` is loaded if the file `vendor/magento/framework/GraphQl/Config/Element/Field.php` doesn't exist, the check doesn't work.
```
PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Magento\Framework\GraphQl\Config\Data\WrappedTypeProcessor::processWrappedType() must implement interface Magento\Framework\GraphQl\Config\Element\FieldInterface, instance of Magento\Framework\GraphQl\Config\Element\Field given, called in /var/www/releases/20200609074425/vendor/magento/framework/GraphQl/Schema/Type/Output/ElementMapper/Formatter/Fields.php on line 125 and defined in /var/www/releases/20200609074425/vendor/magento/framework/GraphQl/Config/Data/WrappedTypeProcessor.php:49
Stack trace:
#0 /var/www/releases/20200609074425/vendor/magento/framework/GraphQl/Schema/Type/Output/ElementMapper/Formatter/Fields.php(125): Magento\Framework\GraphQl\Config\Data\WrappedTypeProcessor->processWrappedType(Object(Magento\Framework\GraphQl\Config\Element\Field), Object(Magento\Framework\GraphQl\Schema\Type\Output\OutputTypeObject))
#1 /var/www/releases/20200609074425/vendor/magento/framework/GraphQl/Schema/Type/Output/ElementMapper/Formatter/Fields.php(143): Magento\Framew in /var/www/releases/20200609074425/vendor/magento/framework/GraphQl/Config/Data/WrappedTypeProcessor.php on line 49
```

**Used versions**
- *Magento Version number*: 2.3.5-p1
- *Open source/Enterprise/B2b*: Enterprise
- *Mollie version number*: 1.14.0

**To Reproduce**
1. Open Postman.
2. Set request method to POST, URL to <magento_url>/graphql.
3. Set request body to.
```
mutation {
  createEmptyCart
}
```
4. Click send button.
5. See error.

**Expected behavior**
The request status is 200, the response is the cart token
```
{
  "data": {
    "createEmptyCart": "4JQaNVJokOpFxrykGVvYrjhiNv9qt31C"}
  }
}
```

**Actual behavior**
A `Fatal error: Uncaught TypeError` occurs

**Comment by daiduongvo18** [comment]
Hi @Frank-Magmodules 

I've submit some information through your contact form.

Thank for your checking!

**Comment by elwin1234** [comment]
Any follow up on this issue? I also encounter the exact same problem as @daiduongvo18. Problem seems to have started since `v1.14.0`.

**Comment by avogel91** [comment]
I have a strong feeling this issue surfaces because the LegacySupport for Magento 2.2. 

The file https://github.com/mollie/magento2/blob/v1.14.0/LegacySupport/GraphQl/Field.php contains a `Field` class that does not implement the `Magento\Framework\GraphQl\Config\Element\FieldInterface`.

I guess the `Field` class is loaded twice. Even though the developer tries to escape loading the class when Magento 2.3 is running

When I comment out the  `/mollie/magento2/LegacySupport/GraphQl/Field.php` file in `vendor/composer/autoload_static.php` and `vendor/composer/autoload_files.php` everything seems to work again. 


I am running Magento 2.3.5CE.

**Comment by Frank-Magmodules** [comment]
Small update; we are testing a fix at this moment and if all goes as planned we will release a new version tomorrow. Thanks for your patience!

**Comment by Frank-Magmodules** [comment]
Hi All, 

Happy to share that we've just released the new [1.1.5](https://github.com/mollie/magento2/releases/tag/v1.15.0) version with a fix for this issue. Thank you again for the detailed reports and patience. We are closing this issue now but please feel free to reopen the issue if this still occurs.
